### ISSUE TITLE
Be consistent with tags; add missing releases

### ISSUE THREAD

**Issue by remram44** [jupyter/notebook#6360]
I noticed that the Git tags are inconsistent, some starting with `v` and some not. I thought you might have changed formats to include `v` after 6.4.1 but you went back to no prefix for 6.4.10, and then back again to `v` for `7.0.0a1`.

It would probably be easier on everyone if you could stick to one format for the tags.

Current tags:

```
v7.0.0a2
v7.0.0a1
6.4.10
v6.4.9
v6.4.8
v6.4.7
v6.4.6
v6.4.5
v6.4.4
v6.4.3
v6.4.2
6.4.1
v6.4.0rc0
v6.4.0
6.3.0
6.2.0
```

<details><summary>older</summary>

```
6.1.6
6.1.5
6.1.4
6.1.3
6.1.2
6.1.1
6.1.0rc1
6.1.0
6.0.3
6.0.2
6.0.1
6.0.0rc1
6.0.0
5.7.9
5.7.8
5.7.7
5.7.6
5.7.5
5.7.4
5.7.3
5.7.2
5.7.13
5.7.12
5.7.11
5.7.10
5.7.1
5.7.0
5.6.0rc1
5.6.0
5.5.0rc1
5.5.0
5.4.1
5.4.0
5.3.1
5.3.0rc1
5.3.0
5.2.2
5.2.1rc1
5.2.1
5.2.0rc1
5.2.0
5.1.0rc3
5.1.0rc2
5.1.0rc1
5.1.0
5.0.0rc2
5.0.0b2
5.0.0b1
5.0.0-rc.1
5.0.0
4.x-test
4.4.1
4.4.0
4.3.2
4.3.1
4.3.0
4.2.3
4.2.2
4.2.1
4.2.0
4.1.0
4.0.6
4.0.5
4.0.4
4.0.3
4.0.2
4.0.1
4.0.0
```

</details>

In addition, [the releases page](https://github.com/jupyter/notebook/releases) is missing 6.4.9 and 6.4.10.

**Comment by jtpio** [comment]
Thanks @remram44 for raising this.

From now one, the tags will be prefixed by `v`, since this is the default for the automated release process. For reference this is the case for `v6.4.11` and the release notes are available here: https://github.com/jupyter/notebook/releases/tag/v6.4.11

Looks like `6.4.10` might have been released manually.

**Comment by jtpio** [comment]
> Looks like `6.4.10` might have been released manually.

Yes that seems to be the case, to address https://github.com/jupyter/notebook/security/advisories/GHSA-m87f-39q9-6f55.

cc @blink1073 probably we can create the GitHub Release on the `6.4.10` tag and use the changelog generation to it shows in the list of releases?

**Comment by blink1073** [comment]
Ah yes, that makes sense.
